ecosystem/sep-0024.md

```diff
@@ -445,16 +446,20 @@ The JSON object contains an entry for each asset that the anchor supports for SE
 * `enabled`: `true` if SEP-6 deposit for this asset is supported
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
-* `fee_fixed`: Optional fixed (flat) fee for deposit. In units of the deposited asset. Leave blank if there is no fee or the fee schedule is complex.
-* `fee_percent`: Optional percentage fee for deposit. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_fixed`: Optional fixed (base) fee for deposit. In units of the deposited asset. This is in addition to any `fee_percent`.  Leave blank if there is no fee or the fee schedule is complex.
+* `fee_percent`: Optional percentage fee for deposit. In percentage points. This is in addition to any `fee_fixed`. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_minimum`: Optional minimum fee in units of the deposited asset.
 
 #### For each withdrawal asset, response contains:
 
 * `enabled`: `true` if SEP-6 withdrawal for this asset is supported
 * `min_amount`: Optional minimum amount. No limit if not specified.
 * `max_amount`: Optional maximum amount. No limit if not specified.
-* `fee_fixed`: Optional fixed (flat) fee for withdraw. In units of the deposited asset. Leave blank if there is no fee or the fee schedule is complex.
-* `fee_percent`: Optional percentage fee for withdraw. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_fixed`: Optional fixed (base) fee for withdraw. In units of the withdrawn asset.  This is in addition to any `fee_percent`.
+* `fee_percent`: Optional percentage fee for withdraw in percentage points. This is in addition to any `fee_fixed`.
+* `fee_minimum`: Optional minimum fee in units of the withdrawn asset.
+
+If fee_fixed or fee_percent is provided, fees are calculated as `(amount * percent fee) + fixed fee = total fee`.  If the fee structure doesn't fit this model, leave them blank and provide the [`/fee`](#fee) endpoint instead.
```

Comment:
Suggested rewording so that all mentions of percent and fixed match the names already used. Adds backticks around named fees also like other mentions of them.
```suggestion
If `fee_fixed` or `fee_percent` are provided, the total fee is calculated as `(amount * fee_percent) + fee_fixed = fee_total`.  If the fee structure doesn't fit this model, leave them blank and provide the [`/fee`](#fee) endpoint instead.
```